templates/solochain/node/src/chain_spec.rs

```diff
@@ -1,38 +1,12 @@
 use sc_service::ChainType;
-use solochain_template_runtime::{AccountId, Signature, WASM_BINARY};
-use sp_consensus_aura::sr25519::AuthorityId as AuraId;
-use sp_consensus_grandpa::AuthorityId as GrandpaId;
-use sp_core::{sr25519, Pair, Public};
-use sp_runtime::traits::{IdentifyAccount, Verify};
+use solochain_template_runtime::WASM_BINARY;
 
 // The URL for the telemetry server.
 // const STAGING_TELEMETRY_URL: &str = "wss://telemetry.polkadot.io/submit/";
```

Comment:
Not introduced here, but lets remove this comment

Reply:
Done here: [cc761df](https://github.com/paritytech/polkadot-sdk/pull/5868/commits/cc761df41aa4232846022e611abe15442d9c92df).